recipe-client-addon/lib/ShieldPreferences.jsm

```diff
@@ -73,22 +73,24 @@ this.ShieldPreferences = {
     let prefValue;
     switch (prefName) {
       // If the FHR pref changes, set the opt-out-study pref to the value it is changing to.
-      case FHR_UPLOAD_ENABLED_PREF:
+      case FHR_UPLOAD_ENABLED_PREF: {
```

Comment:
Huh, are the braces a new eslint thing? I don't mind either way, just surprised...

Reply:
They aren't an eslint thing. I personally prefer them because it creates a scope for `let` and `const`. It doesn't apply here, but doing it consistently makes sense to me.